src/main/webapp/portal/main/partials/sidebar-left.html

```diff
@@ -1,7 +1,7 @@
 <div>
   <nav role="navigation" class="side-bar-nav">
     <div class="hide-sidebar-div">
-      <span class="fa fa-arrow-left hide-sidebar" ng-click="$storage.showSidebar = false" popover="Minimize sidebar" popover-placement="bottom" popover-trigger="mouseenter" popover-popup-delay="600" popover-append-to-body="true"></span>
+      <span class="fa fa-arrow-left hide-sidebar" ng-click="$storage.showSidebar = false" popover="Minimize sidebar" popover-placement="bottom" popover-trigger="mouseenter" popover-popup-delay="600" popover-append-to-body="true" onClick="pushGaEvent('Sidebar','Minimize Sidebar', 'Minimize Sidebar')"></span>
```

Comment:
Subtle tweak: would it be better to for the maximize and minimize events to share an action and differentiate by label, so as to more easily group and compare the maximizing and minimizing?

`"pushGaEvent('Sidebar','Show-hide Sidebar', 'Show Sidebar')"`
`"pushGaEvent('Sidebar','Show-hide Sidebar', 'Hide Sidebar')"`